### PR TITLE
Update devDependencies and fix new issues

### PR DESCRIPTION
**Pull request by XhmikosR**
@Kreeg: I haven't updated playwright-chromium due to that old issue you were facing. I assume you are still facing it with the latest version, right?

The only other deps that are outdated is glob and pngjs. For glob there's no hurry and in fact we better keep it to v7.x for now. pngjs I didn't try to update it, so if you have time feel free to update it :)

### CODE REVIEW

File: test/svg-sprite/config/config.constructor.shape.align.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const yaml = require('js-yaml');
 const SVGSpriterConfig = require('../../../lib/svg-sprite/config.js');
 
 jest.mock('node:fs');
-jest.mock('yaml');
+jest.mock('js-yaml');
```
Comment (XhmikosR) [inline test/svg-sprite/config/config.constructor.shape.align.test.js:10]: I don't quite get how this worked all this time 😛

File: lib/svg-sprite/config.js [hunk 2]
```diff
@@ -324,10 +324,7 @@ module.exports = class SVGSpriterConfig {
         ['padding'].forEach(function(property) {
             let spacing;
 
-            if (!Array.isArray(this.spacing[property])) {
```
Comment (Kreeg) [inline lib/svg-sprite/config.js:327]: Was this eslint's correction?

**Comment by Kreeg** [comment]
Yes. The problem with playwright-chromium still exists. I do not have much time to deal with it and I sure this is a chromium problem rather than a playwright.
I updated pngjs. This was a seamless update, no extra work.

**Review by Kreeg — APPROVED** [review]

File: lib/svg-sprite/config.js (hunk 2 above)
Comment (XhmikosR) [reply lib/svg-sprite/config.js:327]: Yeah, I guess the no negated condition rule I guess added some fixes

Comment (XhmikosR) [reply lib/svg-sprite/config.js:327]: Yeah, I guess the no negated condition rule I guess added some fixes

**Pull request merged**